WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerAdapter.java

```diff
@@ -172,6 +174,14 @@ boolean isVideoUri(Uri uri) {
         return index > -1 && getItemAtPosition(index).isVideo;
     }
 
+    void setAllowMultiSelect(boolean allow) {
+        mAllowMultiSelect = allow;
+    }
+
+    void setShowPhotosOnly(boolean value) {
```

Comment:
Should we change the `title` of the activity when Videos are included in the picker?